stack/full-stack-lab/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM full-stack as base
+
+FROM  lab
+
+USER root
+
+COPY --from=base /opt/config-quick-setup.yaml /opt/
+COPY --from=base "${CONDA_DIR}/envs/aiida-core-services" "${CONDA_DIR}/envs/aiida-core-services"
```

Comment:
That can only work if we manage to install RMQ through conda for the arm64 architecture.

Reply:
That appears to be the case. With https://github.com/aiidalab/aiidalab-docker-stack/pull/279/commits/02449081481cf050ef1e2c1b9bac7b5c6f7b0874 we build (albeit not necessarily test) all image for both architectures.